### DISCUSSION TITLE
TasmoManager / Tom Butcher - who knows anything?

### DISCUSSION THREAD

**Discussion by Pewibe** [arendst/Tasmota#13331] [Q&A] [answered]
Not directly a Tasmota item but related anyhow. Tom Butcher started a project on github named TasmoManager to show and admin the locally installed Tasmota devices. Since May 2020 everything is quite at his github page. His personal homepage is not existing anymore and nobody get any reaction.
Does somebody know what happened to him or TasmoAdmin? Thanks for your hints in advance.

**Comment by ascillato** [comment, marked as answer]
About [TasmoManager](https://github.com/arendst/Tasmota/discussions/11509)-[(Source)](https://github.com/tom-23/TasmoManager) we **don't know** anything. Tom posted some time ago a link here to his tool ( https://github.com/arendst/Tasmota/discussions/11509 ) but besides that post we didn't have any further communications.

About [TasmoAdmin](https://tasmota.github.io/docs/TasmoAdmin/)-[(Source)](https://github.com/reloxx13/TasmoAdmin), the project is **Active**. @reloxx13 is very busy. As soon as he is available again, he will continue. The Home Assistant's [Addon for TasmoAdmin](https://github.com/hassio-addons/addon-tasmoadmin) is also **Active**.

[TDM Tasmota Device Manager](https://tasmota.github.io/docs/Tasmota-Device-Manager/)-[(Source)](https://github.com/jziolkowski/tdm) is a very **Active** project. The Home Assistant's [Addon for TDM](https://github.com/ikifar2012/tasmota-device-manager-addon) is also **Active**.

[Home Assistant's Addon TasmoBackup-(Source)](https://github.com/danmed/TasmoBackupV1) is **Active** too.

[TasmoCompiler](https://tasmota.github.io/docs/Compile-your-build/#online-compilers)-[(Source)](https://github.com/benzino77/tasmocompiler) is **Active** too.

[Tasmotizer](https://tasmota.github.io/docs/Getting-Started/#flashing-tool)-[(Source)](https://github.com/tasmota/tasmotizer) is **Active** too.

[Tasmota Web Installer](https://tasmota.github.io/install/)-[(Source)](https://github.com/tasmota/install) is **Active** too.

[IPS Tasmota Configurator](https://tasmota.github.io/docs/IP-Symcon/#ips-tasmotaconfigurator)-[(Source)](https://github.com/Schnittcher/IPS-Tasmota) is **Active** too.

[Octoprint Tasmota](https://tasmota.github.io/docs/Octoprint/#octoprint-tasmota)-[(Source)](https://github.com/jneilliii/OctoPrint-Tasmota) is **Active** too.

[Microsoft Teams to Tasmota-(Source)](https://github.com/Claus-E/teams2tasmota) is **Active** too.

[Tasmota Device Locator](https://tasmota.github.io/docs/Tasmota-Device-Locator/)-[(Source)](https://github.com/KimNyholm/tasmota-device-locator) is **Offline** from some time now as it requires CORS that is disabled by default in Tasmota due to [security concerns](https://github.com/arendst/Tasmota/issues/6767).

[TasUI](https://tasmota.github.io/docs/TasUI/)-[(Source)](https://github.com/IOTReboot/TasUI) is **Offline** from some time now as it requires CORS that is disabled by default in Tasmota due to [security concerns](https://github.com/arendst/Tasmota/issues/6767).

**Reply by reloxx13** [reply]
we need to take a look for the dockerhub api limit :X

**Reply by Pewibe** [reply]
Looks like an insiders joke - sorry that I can't follow you.

**Reply by sfromis** [reply]
@reloxx13 - Assuming that you mean for download of binaries, you can just skip using Github for that, and instead download from the OTA site 
http://ota.tasmota.com/tasmota/release/
http://ota.tasmota.com/tasmota/
http://ota.tasmota.com/tasmota32/release/
http://ota.tasmota.com/tasmota32/

**Reply by barbudor** [reply]
May be he is refering to the number of download of the docker image from hub.docker.com. 
I know that Docker is limiting the max number of pull (download) a free user can do per day. But I can't see any limit attached to the image itself.
